docs/usage/config-presets.md

```diff
@@ -25,19 +19,17 @@ Shareable config presets can only be used with the JSON format, other formats ar
 !!! info
     We've deprecated the use of a `renovate.json` file for presets as this can cause issues if the repository configuration uses a `renovate.json` file as well.
 
-## Goals of Preset Configs
-
-The main reason for supporting preset configs is to decrease duplication.
-By using a preset config you:
-
-1. Avoid duplicating the same config across all your repositories
-2. Can use someone else's configuration and extend it
+<!-- prettier-ignore -->
+!!! tip
+    Describe what your preset is doing by using the `"description"` field.
+    This way your configuration is self-documenting.
 
-Renovate's configuration is self-documenting, because you can fill in the `"description"` field in all preset configs.
+## Extending from a preset
 
-## Implementation Approach
+To use a preset put it in an `extends` array within your Renovate config.
+Presets can be nested.
 
-In order to achieve these goals, preset configs allow for a very modular approach - preset configs can be as small as a partial package rule or as large as an entire configuration, like an ESLint config.
+QUESTION: Insert `renovate.json` example config with nested preset?
```

Comment:
We should add an example of a nested config preset. This way people can see what's valid nesting, and how it works.

Can you maintainers suggest a good example?

Reply:
What do you mean by nested preset and what's the motivation?

Reply:
I think since we support files and paths for presets, we should deprecate nested presets (docs only note)

Reply:
Preferring one preset per file?

Reply:
yes, would make things easier to explain in docs. 🤗

Reply:
Agreed. We use nested ones in our internal though..

Reply:
> Agreed. We use nested ones in our internal though..

👀 we do? probably the (partial) auto generated presets?

Reply:
> What do you mean by nested preset and what's the motivation?

The _current_ docs mention "nested presets", check https://docs.renovatebot.com/config-presets/:

> Renovate's "config presets" are a convenient way to distribute config for reuse across multiple repositories. It is similar in design to ESLint's shareable configs, and can be used for whole repository configs and for individual rules. They are defined using the `extends` array within config and may also be nested.

That's why I thought we need an example on how to create a nested preset. 😉 

But if we can simplify by dropping support for "nesting" completely, that's even better. 🥳

Reply:
Do you want me to create an issue to "deprecate nested presets in the docs" then?

Reply:
Yes please